docs/get-started/training-pytorch.md

```diff
@@ -1,33 +1,38 @@
 ---
-title: ORT Training with PyTorch
+title: Large Model Training
 parent: Get Started
 nav_order: 12
 ---
 
-# Get started with ORT for Training API (PyTorch)
+# Get started with Large Model Training with ORTModule
 {: .no_toc }
-The ORT Training API is a PyTorch frontend that implements the torch.nn.Module interface.
 
+`ONNX Runtime Training`'s `ORTModule` offers a high performance training engine for models defined using the `PyTorch` frontend. `ORTModule` is designed to accelerate the training of large models without needing to change either the model definition or the training code.
```

Comment:
thanks for making this change!

Reply:
without needing to change either the model definition or the training code -> well you need to change the training code to wrap the model with ORTModule right